src/app/footer/footer.component.css

```diff
@@ -1,3 +1,6 @@
+*{
```

Comment:
I see this is added in some files, what if a new component is added, then the dark mode switch for that will not have a transition? Can we add this to a global styles file? Does it make sense to have it there? What do you think?

Reply:
Great point.
Maybe if a component or several ones might not need it, we can change the property of transition to none using a class for example.
It makes sense to be on global styles file.